runelite-client/src/main/java/net/runelite/client/account/SessionManager.java

```diff
@@ -52,35 +55,49 @@
 @Slf4j
 public class SessionManager
 {
-	private static final File SESSION_FILE = new File(RuneLite.RUNELITE_DIR, "session");
-
 	@Getter
 	private AccountSession accountSession;
 
 	private final EventBus eventBus;
 	private final ConfigManager configManager;
 	private final WSClient wsClient;
+	private final File sessionFile;
 
 	@Inject
-	private SessionManager(ConfigManager configManager, EventBus eventBus, WSClient wsClient)
+	private SessionManager(
+		@Named("sessionfile") String sessionfile,
+		ConfigManager configManager,
+		EventBus eventBus,
+		WSClient wsClient)
 	{
 		this.configManager = configManager;
 		this.eventBus = eventBus;
 		this.wsClient = wsClient;
+
+		Path path = Paths.get(sessionfile);
+		if (path.isAbsolute())
```

Comment:
What if someone is doing `./my-session-file` what is relative path and completely valid use-case? Pretty sure then this check will be incorrect